templates/pages/userform.html.twig

```diff
@@ -91,7 +91,7 @@
                         {% endif %}
                         {% if not options.public or question.getSubField().isPublicFormCompatible() %}
                             {% set sessionData = session('formcreator') %}
-                            {{ question.getRenderedHtml(domain, true, sessionData.data)|raw }}
```

Comment:
Is the `null` part correct or is there an actual `FormAnswer` object that should be referenced here ?